src/components/Helpers/SvgIcon/SvgIcon.tsx

```diff
@@ -7,7 +7,7 @@ interface SvgIconProps {
   handler?: (event: any) => void;
 }
 
-export const SvgIcon = ({ tooltipId, cssClass, children, handler }: SvgIconProps): ReactElement => (
+const SvgIcon = ({ tooltipId, cssClass, children, handler }: SvgIconProps): ReactElement => (
```

Comment:
why this change ? It good to have named export. gives consistance and can import multiple values later on.